components/EmotionDetails.js

```diff
@@ -48,21 +49,27 @@ export default function EmotionDetails({ object, onDeleteEmotion }) {
 }
 
 // Styled Components //
-const StyledLink = styled(Link)`
+const StyledBackLink = styled(Link)`
   font-size: 24px;
   margin-left: 10px;
   text-decoration: none;
   color: #000;
 `;
 
-const StyledContainer = styled.div`
-  font-size: 1rem;
-  margin: 16px 8px;
-  padding: 10px;
-  background-color: #f9f9f9;
-  border: 1px solid #d3d3d3;
-  border-radius: 8px;
-  list-style: none;
+const StyledEditLink = styled(Link)`
+  float: left;
+  margin-right: 20px;
+  margin: 10px;
+  padding: 10px 20px;
+  background-color: #6666ff;
+  color: white;
```

Comment:
I would recommend staying consistent in naming colors. Right now you are using `hex`, `rgb` and named colors. I would stick with either `hex` or `rgb` and use that everywhere.

Reply:
Very good point and we will take it into account when styling the elements.